src/components/taskDetails/index.tsx

```diff
@@ -158,8 +158,12 @@ const TaskDetails: FC<Props> = ({ taskID }) => {
         }
     }
 
-    function getTaskEstimates(timestamp: number | undefined) {
-        return timestamp ? convertTimeStamp(timestamp) : 'NA';
+    function getStartedOn(timestamp: number | undefined) {
+        return timestamp ? convertTimeStamp(timestamp) : 'N/A';
```

Comment:
what if timestamp undefined ...?

Reply:
But that is what I'm checking for . If it is undefined, it goes to the else condition i.e., N/A